string filename
geometry_msgs/Point center
float32 x_width
float32 y_height
float32 resolution
float32 max_z
